leptonai/api/v1/client.py

```diff
@@ -77,7 +77,7 @@ def __init__(
                 " credential string."
             )
         # If workspace_id contains colon, it is a credential that also contains the token.
```

Comment:
nit: add a guard like:

if workspace_id.startswith("https"):
    raise RuntimeError("You have incorrectly entered the URL, not the workspace id, to the workspace id parameter. The workspace id is going to be the WSID section in the URL, like https://dashboard.lepton.ai/workspace/WSID/explore .")

Reply:
Never mind, checked internal testing code and it seems that this won't fix the issue...